rosa_tutorials/rosa-mobb-using-cloudfront-and-waf.adoc

```diff
@@ -0,0 +1,220 @@
+
+[id="preparing_{context}"]
+== Preparing
+
+. Create a cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa create cluster --cluster-name poc-waf --multi-az \
+--region us-east-2 --version 4.7.9 --compute-nodes 3 \
+--machine-cidr 10.0.0.0/16 --service-cidr 172.30.0.0/16 \
+--pod-cidr 10.128.0.0/14 --host-prefix 23
+----
+
+. When the cluster is ready, create a admin user by running the following command:
++
+[source,terminal]
+----
+$ rosa create admin -c poc-waf
+----
+
+. Log in by following the instructions.
+
+. Set the `$EMAIL` and `$DOMAIN` environment variables. For example:
++
+[source,terminal]
+----
+EMAIL=username@example.com
+DOMAIN=waf.tutorial
+----
+
+[id="certificate-and-dns_{context}"]
+== Certificate and DNS
+
+. Create a wildcard certificate by using the `certbot` tool. For example:
++
+[source,terminal]
+----
+certbot certonly --manual \
+--preferred-challenges=dns \
+--email $EMAIL \
+--server https://acme-v02.api.letsencrypt.org/directory \
+--agree-tos \
+--manual-public-ip-logging-ok \
+-d "*.$DOMAIN"
+----
+
+. Create a DNS TXT record by following the `certbot` instructions. The certificate records are saved on your system.
+
+. Set an environment variable for the directory where the certificate records are saved. For example:
++
+[source,terminal]
+----
+CERTS=/etc/letsencrypt/live/waf.tutorial
+----
+
+[id="custom-openshift-domain_{context}"]
+== Custom OpenShift Domain
+
+. Create a project for the custom route by running the following command:
++
+[source,terminal]
+----
+$ oc new-project my-custom-route
+----
+
+. Add the certificates as a secret by running the following command:
++
+[source,terminal]
+----
+$ oc create secret tls acme-tls --cert=$CERTS/fullchain1.pem --key=$CERTS/privkey1.pem
+----
+
+. Create a `CustomDomain` manifest. For example:
++
+[source,terminal]
+----
+cat << EOF | oc apply -f -
+apiVersion: managed.openshift.io/v1alpha1
+kind: CustomDomain
+metadata:
+  name: acme
+spec:
+  domain: $DOMAIN
+  loadBalancerType: Classic <1>
+  certificate:
+    name: acme-tls
+    namespace: my-custom-route
+EOF
+----
+<1> You can optionally change the `spec.loadBalancerType` value to `NLB`. 
+
+. Monitor until your `CustomDomain` object has an endpoint:
++
+[source,terminal]
+----
+$ watch oc get customdomains
+----
+
+[id="aws-waf-and-cloudfront_{context}"]
+== AWS WAF + CloudFront
+
+. Create a WAF rule.
+.. Navigate to https://console.aws.amazon.com/wafv2/homev2/web-acls/new?region=us-east-2. 
+.. Use the Core and SQL Injection rules, and set it as a CloudFront distribution resource type.
+
+. View your WAF by running the following command:
++
+[source,terminal]
+----
+$ aws wafv2 list-web-acls --scope REGIONAL --region us-east-2 | jq .
+----
+
+. Add a certificate to ACM. 
+.. Navigate to https://us-east-2.console.aws.amazon.com/acm/home?region=us-east-1#/importwizard/.
+.. Paste the certificate, key, and certchain from the `certbot` files.
```

Comment:
```suggestion
.. Paste the certificate, key, and certificate chain from the `certbot` files.
```
I think we are implementing changes like this in the second round, but want to flag it.